SR2019/2019-07-02-kit-chase-round-2.md

```diff
@@ -0,0 +1,30 @@
+---
+to: Teams with still outstanding kit
+subject: Outstanding kit - Round 2
+---
+
+Hello!
+
+We sent an email to you back in May stating we required a kit back from you. Unfortunately, we've not heard a response.
+
+The kits are loaned to you fur the duration of the competition year, and as this has ended, we need the kits back, to prepare for future events.
+
+Please securely pack the kit and ship it to the address listed on [our website](https://studentrobotics.org/contact/). Once you have dispatched it, please [let us know](mailto:teams@studentrobotics.org).
```

Comment:
Nowadays most people don't have working mailto links, maybe better to explicitly say "let us know by emailing us at <email>"?

Reply:
In my experience, most people either read the email from the link, or just respond to the email, which will come from that address anyway